src/hotspot/share/services/nmtPreInit.cpp

```diff
@@ -180,8 +179,7 @@ void* NMTPreInit::do_os_malloc(size_t size) {
 // Switches from NMT pre-init state to NMT post-init state;
 //  in post-init, no modifications to the lookup table are possible.
 void NMTPreInit::pre_to_post() {
-  assert(_nmt_was_initialized == false, "just once");
-  _nmt_was_initialized = true;
+  assert(MemTracker::is_initialized() == false, "just once");
```

Comment:
Maybe this should be `!MemTracker::is_initialized()` to be consistent with the rest of the code?

Reply:
Done.